tensorflow/lite/kernels/reverse.cc

```diff
@@ -67,40 +77,59 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
 TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
   const TfLiteTensor* input = GetInput(context, node, kInputTensor);
   const TfLiteTensor* axis_tensor = GetInput(context, node, kAxisTensor);
-  int axis = GetTensorData<int32_t>(axis_tensor)[0];
+  int dimension_size = NumDimensions(input);
+  int axes_size = NumElements(axis_tensor);
+  std::vector<bool> axes(dimension_size, false);
```

Comment:
TF only supports 8 dimensions. Should we just use an std::array here and avoid the vector?